pri/boostdetect.pri

```diff
@@ -51,6 +51,6 @@ exists($$boost_root) {
 
     !contains(LATESTBOOST, installed) {
         message("using boost_1_$${LATESTBOOST}_0")
-        INCLUDEPATH += src/lib/boost_1_$${LATESTBOOST}_0 ../boost_1_$${LATESTBOOST}_0
+        INCLUDEPATH += src/lib/boost_1_$${LATESTBOOST}_0 ../boost_1_$${LATESTBOOST}_0 src/lib/boost_1_$${LATESTBOOST}_0/include ../boost_1_$${LATESTBOOST}_0/include
     }
```

Comment:
Both directories added are relative to the source code directory of Fritzing. How would that work with a system (or brew) installation of boost? Also, for the sanity of this script: The "LATESTBOOST, installed" check is meant to check if we were able to detect a pre-installed version of boost, or one given by the boost_root environment variable.